prometheus-client/src/Prometheus/Metric/Summary.hs

```diff
@@ -22,6 +24,7 @@ import Prometheus.Metric
 import Prometheus.Metric.Observer
 import Prometheus.MonadMonitor
 
+import Control.DeepSeq
```

Comment:
Ultimately up to @fimad, but I'd prefer it if we either imported this qualified or explicitly list the names we want to import.

Reply:
+1 for external modules I find it is easier to read with either qualified or explicit imports.